packages/framework/src/Form/Admin/Order/OrderFormType.php

```diff
@@ -173,10 +190,20 @@ private function createBasicInformationGroup(FormBuilderInterface $builder, Orde
                 'choices' => $this->orderStatusFacade->getAll(),
                 'choice_label' => 'name',
                 'choice_value' => 'id',
+                'choice_attr' => function (OrderStatus $orderStatus) {
+                    return [
+                        'data-js-order-status-type' => $orderStatus->getType(),
+                    ];
+                },
                 'multiple' => false,
                 'expanded' => false,
+                'attr' => [
+                    'class' => 'js-order-status-select',
```

Comment:
I would use a separate data attribute here. 
When we set a class here, we may overwrite the default classes rendered in the template.

```
'attr' => [
    'data-js-order-status-select' => null
```